pynest/nest/tests/test_parrot_neuron.py

```diff
@@ -140,7 +140,7 @@ def test_ParrotNeuronIncomingMultiplicity(self):
 
         source = nest.Create('poisson_generator', params={'rate': rate})
         parrots = nest.Create('parrot_neuron', 2)
-        detect = nest.Create('spike_detector')
+        detect = nest.Create('spike_recorder')
```

Comment:
Maybe use another name than `detect` here.

Reply:
I made it `spike_rec`